packages/tools/devtools/devtools-browser-extension/src/popup/PopupScript.ts

```diff
@@ -28,4 +29,15 @@ browser.tabs.query({ active: true, currentWindow: true }, (tab) => {
 		'To use the Fluid Devtools, open the browser Devtools pane (F12) and click the "Fluid Developer Tools" tab.';
 
 	document.body.append(popupElement);
+	initializePopupView(popupElement).then(() => {
+		if (tabs.length === 0 || tabs[0].id === undefined) {
+			console.error("No active tab found or tab ID is undefined");
+			return;
+		}
+		// Send message to webpage
+		port.postMessage({ tabID: tabs[0].id, data: "Requesting supportedFeatures" });
```

Comment:
Shouldn't we be sending this in `PopupView.tsx` so we can listen for the response?